src/server/app.js

```diff
@@ -1,13 +1,36 @@
-var express = require('express');
-var app = express();
+/* 
 
+	app.js
+
+	Initializes server. Sets up static content. Establishes routes and 
+	their handlers.
+
+	Written by Sam Reaves
+	July 22, 2015
+
+ */
```

Comment:
I should also take this opportunity to mention that WebStorm has great jsdoc support built-in, including auto-adding all the asterisks for you after you type ```/**[tab]``` and autocompleting the default jsdoc commands (you can also add any arbitrary tag you want and style it however you want later when you generate documentation from it - @date, for instance, is not a supported tag, but we can still add it here and display it with fancy styles in the documentation we generate from these tags)